lib/logstash/plugin_mixins/jdbc/statement_handler.rb

```diff
@@ -52,27 +64,18 @@ def perform_query(db, sql_last_value)
 
     def build_query(db, sql_last_value)
       parameters[:sql_last_value] = sql_last_value
-      query = db[statement, parameters]
-      statement_logger.log_statement_parameters(statement, parameters, query)
-      query
+      db[statement, parameters]
```

Comment:
Maybe I'm missing something, but in this case the original `statement_logger.log_statment_parameter` would ave logged the string:

```ruby
@logger.debug("Executing JDBC query", :statement => statement, :parameters => parameters)
```

It's not clear, after removal of the statement logger, who logs the query here.

Reply:
we already rely on query logging from Sequel and allow to customize the level at which SQL gets logged using `sql_log_level => ...` (`info` by default), the query logging already includes parameters the only missing piece is the total record count but that is only relevant when paging.

here's how it looked (before this PR) at debug level, notice the `Executing JDBC query` is redundant information and caused the extra count query:
```
[INFO ][logstash.inputs.jdbc     ][main] (0.000499s) SELECT count(*) AS "count" FROM (SELECT FIRST_NAME, LAST_NAME FROM employee WHERE EMP_NO = 2) AS "t1" LIMIT 1
[DEBUG][logstash.inputs.jdbc     ][main] Executing JDBC query {:statement=>"SELECT FIRST_NAME, LAST_NAME FROM employee WHERE EMP_NO = 2", :parameters=>{:sql_last_value=>2022-05-02 11:29:15 UTC}, :count=>1}
[INFO ][logstash.inputs.jdbc     ][main] (0.000528s) SELECT FIRST_NAME, LAST_NAME FROM employee WHERE EMP_NO = 2
```